pages/platform/kubernetes-k8s/securing-nginx-ingress-cert-manager/guide.en-gb.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Securing Nginx Ingress on OVHcloud Managed Kubernetes with cert-manager
+slug: installing-nginx-ingress
```

Comment:
Hello @scraly , this slug already exists for the Installing Nginx page (https://docs.ovh.com/gb/en/kubernetes/installing-nginx-ingress/)
Please use another slug.

Reply:
Hi, I removed the draft doc, sorry :)